### PR TITLE
go.mod: gvisor.dev/gvisor v0.0.0-20221216231429-a78e892a26d2

### PR DESCRIPTION
**Pull request by AkihiroSuda**
`*stack.PacketBuffer` is now replaced with `stack.PacketBufferPtr`.

Changes: https://github.com/google/gvisor/compare/edc830a43ba6...6ec92611f024

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,7 +24,7 @@ require (
 	golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa
 	golang.org/x/sync v0.1.0
 	golang.org/x/sys v0.2.0
```
Comment (cfergeau) [inline go.mod:26]: This PR conflicts with the `main` branch since `golang.org/x/sys` is  now `v0.3.0`.
Can you rebase it?

**Review by cfergeau — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (AkihiroSuda) [reply go.mod]: done

Comment (cfergeau) [reply go.mod]: Thanks!

**Pull request merged**